FinancialTransactionsApi/V1/Helpers/FileGenerator.cs

```diff
@@ -0,0 +1,130 @@
+using CsvHelper;
+using CsvHelper.Configuration;
+using FinancialTransactionsApi.V1.Boundary.Response;
+using FinancialTransactionsApi.V1.Domain;
+using NodaMoney;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using WkHtmlToPdfDotNet;
+using WkHtmlToPdfDotNet.Contracts;
+
+namespace FinancialTransactionsApi.V1.Helpers
+{
+    public static class FileGenerator
+    {
+        public static byte[] WritePdfFile(List<Transaction> transactions, string name, string period, IConverter converter)
+        {
+
+            var report = new ExportResponse();
+            var data = new List<ExportTransactionResponse>();
+
+            report.FullName = transactions.FirstOrDefault()?.Person?.FullName;
+            report.Balance = Money.PoundSterling(transactions.LastOrDefault().BalanceAmount).ToString();
+            report.BalanceBroughtForward = Money.PoundSterling(transactions.FirstOrDefault().BalanceAmount).ToString();
+            report.StatementPeriod = period;
+            foreach (var item in transactions)
+            {
+
+                data.Add(
+                   new ExportTransactionResponse
+                   {
+                       Date = item.TransactionDate.ToString("dd MMM yyyy"),
+                       TransactionDetail = item.TransactionSource,
+                       Debit = Money.PoundSterling(item.PaidAmount).ToString(),
+                       Credit = Money.PoundSterling(item.HousingBenefitAmount).ToString(),
+                       Balance = Money.PoundSterling(item.BalanceAmount).ToString()
+                   });
+            }
+            report.Data = data;
+            var globalSettings = new GlobalSettings
+            {
+                ColorMode = ColorMode.Color,
+                Orientation = Orientation.Portrait,
+                PaperSize = PaperKind.A4,
+                Margins = new MarginSettings { Top = 10 },
+                DocumentTitle = $"{name} Statement Report"
+            };
+            var objectSettings = new ObjectSettings
```

Comment:
These appear to be configurable items. Should they not be included as environment variables should they change in the future?

Reply:
the configurable item are already set-up on aws